### PR TITLE
Update plugin descriptions

### LINKED ISSUE #27: Update Kubernetes plugin to not include Porters Data
The Kubernetes plugin description is ambiguous. Update to be explicit that it is just for secrets and not storage.

### PR DESCRIPTION
**Pull request by kichristensen**
Update the description to be explicit that the Azure and Kubernetes plugin only is for secrets

Closes #27

### CODE REVIEW

File: plugins/index.json [hunk 1]
```diff
@@ -2,7 +2,7 @@
   {
     "name": "azure",
     "author": "Porter Authors",
-    "description": "Integrate Porter with Azure. Store Porter's data in Azure Cloud and secure your bundle's secrets in Azure Key Vault.",
+    "description": "Integrate Porter with Azure. Let you secure your bundle's secrets in Azure Key Vault.",
```
Comment (sgettys) [inline plugins/index.json]: nit: "Secure your bundle's secrets in Azure Key Vault" reads a little better IMO

File: plugins/index.json [hunk 2]
```diff
@@ -14,7 +14,7 @@
   {
     "name": "kubernetes",
     "author": "Porter Authors",
-    "description": "Integrates Porter with Kubernetes. Store Porter's data and your bundle's secrets in Kubernetes Secret Resources.",
+    "description": "Integrates Porter with Kubernetes. Let you secure your bundle's secrets in Kubernetes Secret Resources.",
```
Comment (sgettys) [inline plugins/index.json]: nit: "Secure your bundle's secrets in Kubernetes Secret Resources" reads a little better IMO

**Review by sgettys — APPROVED** [review]
couple of nits

File: plugins/index.json (hunk 1 above)
Comment (kichristensen) [reply plugins/index.json]: Agree, changed

File: plugins/index.json (hunk 2 above)
Comment (kichristensen) [reply plugins/index.json]: Agree, changed

**Pull request merged**